### DISCUSSION TITLE
QONNX support for "native" PyTorch quantized models

### DISCUSSION THREAD

**Discussion by jmduarte** [fastmachinelearning/qonnx#82] [Ideas]
I was wondering if anyone has looked into supporting conversion to QONNX from "native" PyTorch quantized models: https://pytorch.org/docs/stable/quantization.html

We are working on a project where this could come in handy, and I think in general more people may look to use this kind of quantization.

@maltanar @jicampos @nhanvtran

**Comment by maltanar** [comment]
Last I checked it was problematic to export native PyTorch quantization to ONNX at all, but if there is a pathway to exporting it to `QuantizeLinear - DequantizeLinear` (QDQ) standard ONNX representation we have tools to convert that format to QONNX.
